client/src/components/Edit.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import "../App.css"
+
+export default function Edit() {
+    return (
+        <div>
+            <h1>this is edit page</h1>
+            
+        </div>
+    )
+}
```

Comment:
I don't believe this entire component gets used. Feel free to remove it to cut down on clutter in your directories